### PR TITLE
app-misc/imgurbash2: New package

### PR DESCRIPTION
**Pull request by lucianposton**
Closes: https://bugs.gentoo.org/662620
Package-Manager: Portage-2.3.40, Repoman-2.3.9

### CODE REVIEW

File: app-misc/imgurbash2/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<pkgmetadata>
+	<maintainer type="person">
+		<email>lucianposton@pm.me</email>
+		<name>Lucian Poston</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+</pkgmetadata>
```
Comment (mgorny) [inline app-misc/imgurbash2/metadata.xml]: Optionally: add remote-id thingie for github.

File: app-misc/imgurbash2/imgurbash2-2.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+SLOT="0"
+IUSE="X"
+
+RDEPEND="
+	net-misc/curl
+	X? ( || ( x11-misc/xclip x11-misc/xsel ) )
+"
+
+src_install() {
+	default
```
[the remark this answers is not in the document]
Comment (lucianposton) [reply app-misc/imgurbash2/imgurbash2-2.1.ebuild]: done

File: app-misc/imgurbash2/metadata.xml (hunk 1 above)
Comment (lucianposton) [reply app-misc/imgurbash2/metadata.xml]: done

**Comment by mgorny** [comment]
Thank you, great work! I'm going to merge it in a minute.

**Pull request closed without merging**